src/Accounts/Accounts/Utilities/CommandMappings.json

```diff
@@ -4180,6 +4180,7 @@
       "Add-AzVpnClientRevokedCertificate": {},
       "Add-AzVpnClientRootCertificate": {},
       "Approve-AzPrivateEndpointConnection": {},
+      "Convert-AzNetworkWatcherClassicConnectionMonitor": {},
```

Comment:
No need to update this file cause we would overwrite it when release.

Reply:
removed this update.